msteams-platform/concepts/bots/bots-files.md

```diff
@@ -0,0 +1,215 @@
+# Send and receive files through your bot
+
+>**Note:** This feature is currently available in Developer Preview.
+
+Your bot can send and receive files with users in the personal (1:1) context. You can use this to implement expense reporting, image recognition, file archival, e-signatures, and other scenarios involving direct manipulation of file content. Files shared in Teams typically appear as cards, and allow rich in-app viewing. In channels, you can also post messages with card attachments referencing existing SharePoint files.
+
+## Overview
+
+One way of doing this is using the Microsoft Graph APIs for OneDrive and SharePoint, [as documented here](https://docs.microsoft.com/en-us/onedrive/developer/rest-api/). This method requires obtaining ongoing access to the user's OneDrive folder through standard OAuth2 authorization flow. This documentation describes a simple alternative mechanism if you only need to send file content as a result of direct user interaction, e.g. sending a message. This API is provided as part of the Microsoft Teams Bot Platform.
+
+## Receiving files in 1:1 chat
+When a user sends a file to your bot, the file is first uploaded to the user's OneDrive for Business storage. Your bot will then receive a message activity notifying you of the user upload. The activity will contain file metadata, such as its name and the content URL, and the content URL's expiry time (e.g. 24 hours). You can directly read from this URL to fetch its binary content.
```

Comment:
The reference to the "contentUrl" below is confusing, because the bot gets the content from **downloadUrl** not contentUrl. A table like you have below for `FileCard` would be helpful.

Reply:
Clarified the language